app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/model/DummyCategory.kt

```diff
@@ -12,7 +12,4 @@ import kotlinx.parcelize.Parcelize
  */
 @Parcelize
 data class DummyCategory(override val id: String, override val name: String): Category {
-    override fun equals(other: Any?): Boolean {
```

Comment:
Why do you want to remove this?

Reply:
because it is a data class, so it already has the equal method built in. And if not we need to overwrite hashCode too.

Reply:
Oh I see okay